### PR TITLE
Merge : inpect.yml into push.yml

### LINKED ISSUE #1696: Feature Request : Github-actions - merge inspect.yml into push.yml
**Is your feature request related to a problem? Please describe.**

- The current GitHub Actions configuration in the API repository involves two separate YAML files, namely inspect.yml and push.yml, both triggered on every push. To streamline and simplify the workflow, we propose consolidating the code from inspect.yml into push.yml and subsequently removing inspect.yml.

**Describe the solution you'd like**

* In the API repo, the commands in the inspect.yml file are invoked with every push.
* We have a separate push.yml file that is also invoked with every push.
* We need to move the YAML code in the inspect.yml to the push.yml file and delete the inspect.yml file
* When this is done, the name of the job needs to be changed from test to something indicative of doing introspection testing.
* It the migrated YAML code needs to:
be dependent on the Push-Workflow job in the push.yml file
ensure that the inspect.yml file's code automatically adjusts to the branch
* This will mean that developers will only need to edit one file for all push related changes to our GitHub actions

**Describe alternatives you've considered**

**Approach to be followed (optional)**
A clear and concise description of approach to be followed.

**Additional context**
- This adjustment aims to centralize all push-related changes into a single file (push.yml), making it more straightforward for developers to manage and update GitHub Actions configurations. Additionally, the workflow's naming conventions will be improved to better reflect its purpose.

**Potential internship candidates**
Please read this if you are planning to apply for a Palisadoes Foundation internship https://github.com/PalisadoesFoundation/talawa/issues/359

### PR DESCRIPTION
**Pull request by aialok**
**What kind of change does this PR introduce?**

- The current GitHub Actions configuration in the API repository involves two separate YAML files, namely inspect.yml and push.yml, both triggered on every push. To streamline and simplify the workflow, we propose consolidating the code from inspect.yml into push.yml and subsequently removing inspect.yml.

**Issue Number:**

Fixes #1696 

**Did you add tests for your changes?**

- No

**Snapshots/Videos:**

**If relevant, did you update the documentation?**

**Summary**

**Does this PR introduce a breaking change?**

**Other information**

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-api/blob/master/CONTRIBUTING.md)?**

### CODE REVIEW

File: .github/workflows/push.yml [hunk 1]
```diff
@@ -165,3 +165,35 @@ jobs:
         user_name: '${{github.actor}}'
         commit_message: 'Talawa API docs updated'
       
+  Check-schema:
+    name: Check Schema
```
Comment (palisadoes) [inline .github/workflows/push.yml:169]: This should depend on the success of `Push-Workflow` tests

**Review by palisadoes — CHANGES_REQUESTED** [review]
See comment

**Comment by palisadoes** [comment]
@noman2002 please review

**Comment by aialok** [comment]
@palisadoes, Please have a look.

**Review by palisadoes — APPROVED** [review]

**Comment by palisadoes** [comment]
This has to be reverted. It's failing. https://github.com/PalisadoesFoundation/talawa-api/actions/runs/7603568839/job/20705612488

**Pull request merged**